Generate_graph.py

```diff
@@ -86,13 +107,15 @@ def add_exposure(G, main_dir, country_iso3):
         'ADM2_EN',
         'ADM1_PCODE',
         'ADM2_PCODE',
+        'tot_sad',
         'group_pop_f',
         'group_pop_m'
     ]
     exposure = exposure[columns]
     # Rename some
     rename_dict = {
         'ADM2_EN': 'name',
+        'tot_sad': 'population',
```

Comment:
I will make a PR to add population and pop density computed from summing the age / gender data, so you can leave this out (as well as line 110)